console/billing/how-to/use-billing-alerts.mdx

```diff
@@ -0,0 +1,66 @@
+---
+meta:
+  title: How to use billing alerts
+  description: This page explains how to use billing alerts
+content:
+  h1: How to use billing alerts
+  paragraph: This page explains how to use billing alerts
+tags: billing-alert alerts billing budget 
+dates:
+  validation: 2024-02-14
+  posted: 2024-02-14
+categories:
+  - billing
+---
+
+The **billing alerts** within the Scaleway console allow you to oversee your expenses efficiently. This feature allows you to create alerts that notify users when predefined budget thresholds are met. Your alerts can be seamlessly delivered through SMS, email, or API webhook channels.
+
+Your budget, denominated in euros, represents the limit for your expenses. Simultaneously, the threshold corresponds to a predetermined percentage of this budget. You have the autonomy to define both the budget and the threshold within the console interface.
+
+**For example:**
+
+  - If you have defined a budget of €1,000, you may configure an email notification to be sent once you have consumed 50% of this limit. In this case, once your consumption reaches €500 you will receive an email informing you the limit threshold has been reached.
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+</Message>
+
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```

Comment:
Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking Go to Billing above the Current consumption area.